### DISCUSSION TITLE
Storing Parameter as a json

### DISCUSSION THREAD

**Discussion by tarun067** [aws/aws-dotnet-extensions-configuration#143] [General]
I have stored a json in AWS parameter store
eg:
Path: env/parameter/BaseParam
Key: Param1
value
{ "x":"TestValue",
  "Y":"TestValue2"}

Getting these param as follow 
 builder.Configuration.AddSystemsManager(source =>
            {
                source.Optional = true;
                source.Path = $"/" + env + "/Parameter";
                source.ReloadAfter = TimeSpan.FromSeconds(200000);
                source.ParameterProcessor = new JsonParameterProcessor();
            }).Build();
It is not loading parameter into configuration provider
Tried getting value by this seems like this one is not working
 builder.Configuration.AddSystemsManager(source =>
            {
                source.Optional = true;
                source.Path = $"/" + env + "/Parameter";
                source.ReloadAfter = TimeSpan.FromSeconds(200000);
            }).Build();
Any solution to get a json from paramter store?

**Comment by ashishdhingra** [comment]
@tarun067 Please have a look at the issue https://github.com/aws/aws-dotnet-extensions-configuration/issues/123. Your scenario appears similar and the support for JsonParameterProcessor was fixed as part of this issue.
